bindings/nodejs/examples/01_get_info.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright 2021-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+import { Client, initLogger } from '@iota/client';
+
+// Run with command:
+// node ./dist/01_get_info.js
+
+// In this example we will get information about the node
+async function run() {
+    initLogger({
```

Comment:
Just thinking but logger and if we can remove the duplication across examples because we define it again in every example.

Reply:
I've resolved this by setting a default config for initLogger.